src/main/kotlin/com/terraformation/backend/tracking/mapbox/MapboxService.kt

```diff
@@ -66,16 +73,106 @@ class MapboxService(
 
         responsePayload.token
       } catch (e: ClientRequestException) {
-        log.error("Mapbox token request failed with HTTP ${e.response.status}")
+        log.error("Mapbox tile request failed with HTTP ${e.response.status}")
+        log.info("Mapbox tile error response payload: ${e.response.bodyAsText()}")
```

Comment:
This is still a token request, right? Shouldn't this say "token request" instead of "tile request"?